meetings/bordeaux2018/index.md

```diff
@@ -24,23 +24,56 @@ The JuMP-dev workshop is a great opportunity to meet other people working on and
 
 ## Location
 
-The workshop will be held at the Institut de Mathématiques de Bordeaux, University of Bordeaux.
+The workshop will be held at the [Institut de Mathématiques de Bordeaux, University of Bordeaux](https://goo.gl/maps/PaeJbcRC58K2). If you are not staying within walking distance, the Talence campus is a short tram ride from the city centre.
```

Comment:
Maybe say more explicitly that there are options to stay in Talence (walking distance) or city center (short tram ride).